libs/fluxc/build.gradle

```diff
@@ -24,6 +24,9 @@ android {
             schemaDirectory "$projectDir/schemas"
         }
         consumerProguardFiles 'consumer-rules.pro'
+
+        buildConfigField "boolean", "ENABLE_WPAPI", "false"
+        buildConfigField "boolean", "WP_ENABLE_DATABASE_DOWNGRADE", "true"
```

Comment:
I understand the reasoning, but I'd prefer to not change the logic of the runtime app in this PR. If you think this should be _changed_, I'd vote to prepare another PR/issue and discuss it with the team. In this PR, I'd focus only on migration.

Reply:
> Note, this, along with the fact that product team will only have that enabled if they copy-paste this specific file from gradle.properties-example to gradle.properties, makes it seems less of a problem, that is, comparing to now having that enabled by default no matter, and for all.

In the past, product team had this problem as well, as this property was loaded from `gradle.properties-example` if `gradle.properties` file was not found: https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/5dfbbc993d2472f6aad2b9f357021506ecfe7152/fluxc/build.gradle#L52

Reply:
> ...but I'd prefer to not change the logic of the runtime app in this PR.

But, my point is exactly that, with this change we are actually changing the logic of the runtime app in this PR, is it not, am I am missing something @wzieba ? 🤔 

Put it another way:
- **Previously**: If you don't copy-paste `gradle.properties-example` into `gradle.properties` on the `fluxc` module, then you wouldn't get the `WP_ENABLE_DATABASE_DOWNGRADE` as `true`, instead it will default to `false`. I am not expecting anyone cloning/working this app to go and copy-paste `gradle.properties-example` into `gradle.properties` on the `fluxc` module level, unless they need to for testing purposes, nor we do such a thing on CI.
- **Now**: Having the `WP_ENABLE_DATABASE_DOWNGRADE` as `true` on the `build.gradle` level, it will default to `true`, both locally and on CI. When this PR will get merge, every developer will start having a database that has downgrade enabled by default.

Reply:
> In the past, product team had this problem as well, as this property was loaded from gradle.properties-example if gradle.properties file was not found: https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/5dfbbc993d2472f6aad2b9f357021506ecfe7152/fluxc/build.gradle#L52

Oh, interesting, I missed this part, that the build script will actually default to `gradle.properties-example` if `gradle.properties` is missing and load those values instead, interesting, then my comment above is wrong and you're right... 🤔